backend/composer.json

```diff
@@ -38,6 +47,27 @@
     "spatie/laravel-ignition": "^1.2"
   },
   "scripts": {
+    "intern": [
+      "@intern-help"
+    ],
+    "intern-help": [
+      "sh -c 'cd static && php index.php --help --composer'"
+    ],
+    "intern-build": [
+      "rm -rf ../public/api",
+      "sh -c 'cd static && php index.php --build'"
+    ],
+    "intern-populate": [
+      "sh -c 'cd static && php index.php --populate'"
+    ],
+    "intern-fetch": [
+      "sh -c 'cd static && php index.php --fetch'"
+    ],
+    "intern-purge": [
+      "rm -rf ../public/api"
```

Comment:
As far as i can see there is no `static` directory on your branch. Or may I missing something? https://github.com/blumilksoftware/internships/tree/148-db-preparation/backend

Reply:
References to the old static backend were added by mistake while resolving merge conflicts. They are obviously unnecessary and will be removed.